python/pyspark/sql/dataframe.py

```diff
@@ -1330,6 +1333,9 @@ def toDF(self, *cols):
     def toPandas(self):
         """Returns the contents of this :class:`DataFrame` as Pandas ``pandas.DataFrame``.
 
+        Note that this method should only be used if the resulting array is expected
```

Comment:
this is super minor, but toPandas results in a Panda's DataFrame rather than an array